modules/vrtcalBidAdapter.js

```diff
@@ -148,7 +150,34 @@ export const spec = {
     );
     ajax(winUrl, null);
     return true;
+  },
+
+  getUserSyncs: function(syncOptions, serverResponses, gdprConsent = {}, uspConsent = '', gppConsent = {}) {
+    const syncs = [];
+    const gdprFlag = `&gdpr=${gdprConsent.gdprApplies ? 1 : 0}`;
+    const gdprString = `&gdpr_consent=${encodeURIComponent((gdprConsent.consentString || ''))}`;
+    const usPrivacy = `&us_privacy=${encodeURIComponent(uspConsent)}`;
+    const gpp = gppConsent.gppString ? gppConsent.gppString : '';
+    const gppSid = Array.isArray(gppConsent.applicableSections) ? gppConsent.applicableSections.join(',') : '';
+    var vrtcalSyncURL = ''
```

Comment:
For consistency, can you change this `var` to a `let`?

Reply:
Adjustment made